.travis.yml

```diff
@@ -34,8 +34,8 @@ install:
   - pip install python-ldap
   - pip install unidecode
   - pip install validate_email
-  - printf '[options]\n\nrunning_env = dev' > ${HOME}/.openerp_serverrc
-  - ln -s server_environment_files_sample ./server_environment_files
+  - printf '[options]\n\nrunning_env = dev\n' > ${HOME}/.openerp_serverrc
+  - ln -s ~/build/OCA/server-tools/server_environment_files_sample ~/build/OCA/server-tools/server_environment_files
```

Comment:
As said by @moylop260, this should be `TRAVIS_BUILD_DIR` (with the proper shell notation).